app/src/main/java/org/simple/clinic/drugstockreminders/CompleteDrugStockReportPayload.kt

```diff
@@ -0,0 +1,17 @@
+package org.simple.clinic.drugstockreminders
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import java.util.UUID
+
+@JsonClass(generateAdapter = true)
+data class CompleteDrugStockReportPayload(
```

Comment:
Any reason why it's "Complete" drug stock report?. Should we rename this to just `DrugStockReportPayload` instead? 🤔